extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -172,13 +181,18 @@ export class ConsentUI {
         if (!this.ui_) {
           return;
         }
-        toggle(this.ui_, true);
+
         if (!this.isPostPrompt_) {
+
+          this.maybeShowOverlay_();
+
           // scheduleLayout is required everytime because some AMP element may
           // get un laid out after toggle display (#unlayoutOnPause)
           // for example <amp-iframe>
           this.baseInstance_.scheduleLayout(this.ui_);
         }
+
+        toggle(this.ui_, true);
```

Comment:
I'm pretty new to the AMP codebase, but I think moving toggle to down here might be what's causing #20548